czicompress/README.md

```diff
@@ -38,8 +38,8 @@ The tool is based on [libczi](https://github.com/ZEISS/libczi.git).
 ## Download
 We have not yet published any Releases. Meanwhile, you can download binaries from the artifacts of the latest [Build workflow run](https://github.com/ZEISS/czicompress/actions/workflows/czicompress_cmake.yml?query=branch%3Amain).
 Click on the topmost successful run and download the binaries for your platform:
-* czicompress-windows-64-release-msvc-package-off for Windows
-* czicompress-ubuntu-release-llvm-package-off for Linux
+* czicompress-windows-64-msvc-package-on-Release for Windows
+* czicompress-ubuntu-llvm-package-off-Release for Linux
```

Comment:
Shouldn't this be
```suggestion
* czicompress-ubuntu-package-off-Release for Linux
```

Reply:
Good catch. Was obviously wrong before already. Will fix.